server/modules/category/repository/categoryRepository.js

```diff
@@ -15,6 +8,13 @@ import {
   deleteSurveySchemaTableProp,
   dbTransformCallback,
 } from '@server/modules/survey/repository/surveySchemaRepositoryUtils'
```

Comment:
we should use all relative imports or absolute imports I think... not both!

Reply:
i didnt touch that i think that the autoformat move the things